docs/operator-manual/policy-servers/02-private-registry.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_label: "Using private registry"
+title: ""
+---
+
+# Configuring Policy Servers to use private registry
+
+It is possible to specify and configure the credentials used to authenticate
+into a private registry where your policies are stored.
```

Comment:
This sentence is a little ambiguous. We need to be more explicit about what/who we are authenticating.

Reply:
I've rewrote the paragraph. Can you review again, please?